programmers/난이도별/level01.시저_암호/SSSOy.java

```diff
@@ -0,0 +1,18 @@
+class Solution {
+    public String solution(String s, int n) {
+        String answer = "";
+        
+        char str[] = s.toCharArray();
+    
+        for(char ch : str) {
+            for(int j = 0; j < n; j++) {
+                ch = (char)((ch == ' ')? ch :
+                           (ch == 'z' || ch == 'Z')? ch - 25 : ch + 1);
+            }//for
+            answer += Character.toString(ch);
+        }//foreach
+        
+        return answer;
```

Comment:
```suggestion
        return s.chars()
                .map(c -> Character.isLowerCase(c) ? (char)('a' + (c + n - 'a') % 26) :
                          Character.isUpperCase(c) ? (char)('A' + (c + n - 'A') % 26) :
                          c)
                .mapToObj(c -> Character.toString((char) c))
                .collect(Collectors.joining(""));
```
Stream을 이용하면 이렇게 표현할 수 있습니다!

Reply:
와 Stream을 사용하면 많은 코드를 간단하게 사용할 수 있군요!!
알려주셔서 감사합니다~ Stream 공부해보겠습니당!!!!😄